domains/prayas.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "prayasdev"
+    },
+    "record": {
+        "RECORDTYPE": "RECORDCONTENT"
```

Comment:
File does not follow proper schema. See docs for more information: https://docs.is-a.dev/